SDWebImage/SDWebImageManager.m

```diff
@@ -151,7 +151,7 @@ - (BOOL)diskImageExistsForURL:(NSURL *)url {
                         completedBlock(nil, error, SDImageCacheTypeNone, finished);
                     });
 
-                    if (error.code != NSURLErrorNotConnectedToInternet) {
+                    if (error.code != NSURLErrorNotConnectedToInternet && error.code != NSURLErrorCancelled && error.code != NSURLErrorTimedOut) {
```

Comment:
@bpoplauschi  This condition should be OR not AND?
error.code != NSURLErrorNotConnectedToInternet || error.code != NSURLErrorCancelled || error.code != NSURLErrorTimedOut